test/test.h

```diff
@@ -18,7 +19,7 @@ class Test {
   static Test* firstTest;
 };
 
-#define TEST(x) static struct Test##x : public Test { Test##x() : Test(#x) {} void Run() override; } __inst_##x; void Test##x::Run()
+#define TEST(x) static struct Test##x : public Test { Test##x() : Test(#x) {} void Run() OPTIONAL_FEATURE_OVERRIDE; } __inst_##x; void Test##x::Run()
```

Comment:
Just remove it then. I prefer not to have build-system magic affect the source code so the code is readable on its own.